web/src/messages_overlay_ui.ts

```diff
@@ -28,13 +40,17 @@ export function focus_on_sibling_element(context) {
         elem_to_be_focused_id = $prev_row.attr(context.id_attribute_name);
     }
 
-    const new_focus_element = get_element_by_id(elem_to_be_focused_id, context);
+    const new_focus_element = get_element_by_id(elem_to_be_focused_id ?? "", context);
     if (new_focus_element[0] !== undefined) {
```

Comment:
Checks like this on `JQuery` should better be written as `$obj.length > 0`.
Here is a complete list of checks that can be fixed in a separate commit:
```
eslint    |    37:9   warning  Unnecessary conditional, the types have no overlap                        @typescript-eslint/no-unnecessary-condition
eslint    |    39:16  warning  Unnecessary conditional, the types have no overlap                        @typescript-eslint/no-unnecessary-condition
eslint    |    44:9   warning  Unnecessary conditional, the types have no overlap                        @typescript-eslint/no-unnecessary-condition
eslint    |   124:9   warning  Unnecessary conditional, value is always truthy                           @typescript-eslint/no-unnecessary-condition
eslint    |   159:9   warning  Unnecessary conditional, the types have no overlap                        @typescript-eslint/no-unnecessary-condition
eslint    |   162:9   warning  Unnecessary conditional, the types have no overlap                        @typescript-eslint/no-unnecessary-condition
```